test/stack-test.js

```diff
@@ -76,12 +68,16 @@ describe('Stack interactions', function () {
     return assert.notEqual(stack.cards.length, stack2.cards.length);
   });
 
-  it('cards can be removed from a stack instance', function () {
+  it('a new stack instance contains two cards', function () {
```

Comment:
Why are blank `Stack` instances assumed to contain two cards? If you use the command `let stack = new Stack();`, you get an instance of the `Stack` class, but no card drop event occurred to trigger that creation and therefore there are no cards that are being automatically added to the new `Stack`.